portfolio/src/main/webapp/script.js

```diff
@@ -12,6 +12,35 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
+
+/* toggles the drop down text */
+function toggleDropDown(id) {
+  document.getElementById(id).classList.toggle("show");
+}
+
+/* display the content corresponding to the tab-button clicked */
+function displayGallery(event, tab_name) {
+  // hide all tab contents
+  var tab_contents = document.getElementsByClassName("tab-content");
```

Comment:
I believe the naming convention for javascript is to "camel casing" where you capitalize the first letter of every word after the first word, and don't use underscore.
Example: thisIsAVariableName

Reply:
Got it. This part is deleted later though.